client/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+import { Flex } from '../Flex';
+import { Text } from '../Text';
+
+import { StyledAvatarImage } from './Avatar.styled';
+
+type AvatarProps = {
+  picture: string | null;
+  name: string;
+};
+
+const DEFAULT_AVATAR_URL = 'https://ahmadda-dev.s3.ap-northeast-2.amazonaws.com/profile_avatar.png';
+
+export const Avatar = ({ picture, name }: AvatarProps) => {
+  const [currentSrc, setCurrentSrc] = useState(picture || DEFAULT_AVATAR_URL);
+
+  const handleImageError = () => {
+    if (currentSrc !== DEFAULT_AVATAR_URL) {
+      setCurrentSrc(DEFAULT_AVATAR_URL);
+    }
+  };
+
+  return (
+    <Flex dir="row" gap="12px" alignItems="center">
+      <StyledAvatarImage
+        src={currentSrc}
```

Comment:
p1.
상태 없이 요롷게 해결 가능할 것 같아요~!

```suggestion
        src={picture ?? DEFAULT_AVATAR_URL}
```